Tipstaff/Presenters/GraphPresenter.cs

```diff
@@ -22,8 +22,8 @@ public GraphData GetGraphData(GraphPeriod gp)
             DateTime startDate = new DateTime();
             var graphPeriod = gp;
             var all = _tipstaffRecordPresenter.GetAll();
-            var warants = all.Select(x => x.Descriminator == "Warant") as List<Warrant>;
-            var childAbductions = all.Select(x => x.Descriminator == "ChildAbduction") as List<ChildAbduction>;
+            var warants = all.Select(x => x.Discriminator == "Warant") as List<Warrant>;
```

Comment:
I've just noticed this - this code has never worked, but they've never reported it. it's comparing to string 'Warant' instead of "Warrant"! Let's change this